src/LazyLoadImage/LazyLoadImage.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import { Meta, StoryFn } from "@storybook/react";
+
+import LazyLoadImage from "./LazyLoadImage";
+import { LazyLoadImageProps } from "./LazyLoadImage.types";
+import React from "react";
+
+/**
+ * Wrapper to lazy load an image
+ */
+const meta: Meta<typeof LazyLoadImage> = {
+  component: LazyLoadImage,
+  title: "LazyLoadImage/LazyLoadImg"
+};
+export default meta;
+
+const Template: StoryFn<LazyLoadImageProps> = args => {
+  return (
+    <>
+      <LazyLoadImage
+        {...{
+          ImgProps: {
+            style: { aspectRatio: "2/1", objectFit: "contain", width: "80%" }
```

Comment:
If i wrap the image in a div with fixed height and width with this style it going to look like below. 
![Screenshot 2024-09-18 112335](https://github.com/user-attachments/assets/a7f2b9fa-e31d-4e6b-be6e-2045f76ceccc). We need the image fit to the outer div.

Reply:
I don't think this is something that we should handle inside the component. This is purely down to styling, if you were passing different styling props, then it would work the way you want to. I thought the idea here is that we use this component just like we would an `<img>`, so the styling is as much a blank slate as possible, we expose the props, and those are freely changeable in VIRTO.

Reply:
I agree, this component is simple a wrapper for <img>. This component shouldn't be making any decisions on styling, other than what it needs for the display hiding during load.